src/blocks/content-helper/current-post-details/component.tsx

```diff
@@ -53,19 +54,7 @@ function CurrentPostDetails() {
 	}, [] );
 
 	if ( error ) {
-		// "Soft" error for which we do not want to show an "Error:" prefix.
-		if ( 'string' === typeof error ) {
-			return <p>{ error }</p>;
-		}
-
-		// Error coming from apiFetch.
-		if ( error?.message ) {
-			return <p>{ __( 'Error:', 'wp-parsely' ) } { error.message }</p>;
-		}
-
-		// Error coming from the WordPress REST API.
-		const errorMessage = JSON.stringify( error ).match( /\[\"(.*?)\"\]/ )[ 1 ];
-		return <p>{ __( 'Error:', 'wp-parsely' ) } { errorMessage }</p>;
+		return <p>{ error.message }</p>;
```

Comment:
How are we going to include `ErrorHint` here?

Reply:
The updated code should take care of that.